lib/chef/resource/apt_repository.rb

```diff
@@ -120,7 +120,7 @@ class AptRepository < Chef::Resource
 
       property :components, Array,
         description: "Package groupings, such as 'main' and 'stable'.",
-        default: lazy { [] }
+        default: lazy { [] }, default_description: "If using a PPA this will be 'main'."
```

Comment:
The "this" in the default description needs to be clarified in the default description unless there is other content providing context elsewhere. Grouping? Component? Property?

Reply:
Updated